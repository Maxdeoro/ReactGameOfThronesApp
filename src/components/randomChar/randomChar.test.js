
import * as React from 'react';
// import {render,screen} from '@testing-library/react';
import renderer from 'react-test-renderer';

import RandomChar from './randomChar';

//JEST
describe('Testing<RandomChar/>', () => {
    it ('RandomChar have rendered correctly', () => {
        const char = renderer.create(<RandomChar/>).toJSON(); 
        expect(char).toMatchSnapshot();
    })
})

// it('RandomChar have rendered correctly', () => {
//     const component = renderer.create(<RandomChar/>);
//     let tree = component.toJSON();
//     expect(tree).toMatchSnapshot();
// })


//@testing-library/react

// describe('RandomChar', () => {
//     it('RandomChar have rendered correctly', () => {
//         render(<RandomChar/>);

//         // expect(screen.queryByText('momondo')).toBeNull();
        
//     })
// });