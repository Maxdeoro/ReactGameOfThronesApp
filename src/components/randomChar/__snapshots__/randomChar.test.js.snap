// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing<RandomChar/> RandomChar have rendered correctly 1`] = `
<div
  className="random-block rounded"
>
  <div
    className="loadingio-spinner-spinner-higd5ofcc3b"
  >
    <div
      class="ldio-srxxrr8e3k9"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
